Makefile.include

```diff
@@ -559,7 +559,10 @@ flash-only: $(FLASHDEPS)
 preflash: $(BUILD_BEFORE_FLASH)
 	$(PREFLASHER) $(PREFFLAGS)
 
-term: $(filter flash, $(MAKECMDGOALS)) $(TERMDEPS)
+termdeps: $(TERMDEPS)
+	@true
+
+term: $(filter flash, $(MAKECMDGOALS)) termdeps
```

Comment:
I would prefer to keep `TERMDEPS` here as there is not reason to depend on the named target.